packages/app/src/app/overmind/actions.ts

```diff
@@ -253,7 +253,7 @@ type ModalName =
   | 'share'
   | 'signInForTemplates'
   | 'userSurvey'
-  | 'liveSessionEnded'
+  | 'liveSessionConfirm'
```

Comment:
Good find!

Reply:
I found it weird that it didn't throw an error... I thought typescript would catch the strings being equal. 🤔